lib/rules/no-restricted-imports.js

```diff
@@ -104,16 +107,20 @@ const arrayOfStringsOrObjectPatterns = {
                     }
                 },
                 additionalProperties: false,
-                required: ["group"],
                 not: {
                     anyOf: [
+                        { required: ["group", "regex"] },
```

Comment:
```suggestion
```

I think this restriction is now already covered by the `oneOf` below.

Reply:
yes, updated.